packages/sitecore-jss-nextjs/package-lock.json

```diff
@@ -1,10715 +1,6696 @@
 {
```

Comment:
Looks like something's not quite right with `sitecore-jss-nextjs`. I'm seeing:

- A merge conflict on the lock file
- Errors on ` npm run build`
- The lock file indentation formatting changed, which makes it difficult to see what actually changed

Reply:
These should be resolved now.